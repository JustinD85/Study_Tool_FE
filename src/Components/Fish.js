import React, { Component } from 'react';

class Fish extends Component {


  render() {
    const { word, index, setUserScore } = this.props;
    return (
      <span className="Fish" data-id={word}>
        <p className="fish-word">{word}</p>
        <img
          src={`./fish${index}.svg`}
          className="fish-svg"
          onClick={setUserScore}
        />
      </span>
    )
  }
}

export default Fish;