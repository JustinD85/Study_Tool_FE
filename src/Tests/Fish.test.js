import React from 'react';
import Fish from '../Components/Fish';
import { shallow } from 'enzyme';

describe('<Fish/>', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Fish />
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});