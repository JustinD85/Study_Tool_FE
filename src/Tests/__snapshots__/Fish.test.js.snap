// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Fish/> should match the snapshot 1`] = `
<span
  className="Fish"
>
  <p
    className="fish-word"
  />
  <img
    className="fish-svg"
    src="./fishundefined.svg"
  />
</span>
`;
